test_chime.py

```diff
@@ -1,12 +1,31 @@
+import importlib
+import pathlib
+import platform
 import subprocess
+import tempfile
+import textwrap
 import time
 import typing
 
+import _pytest.monkeypatch
 import pytest
 
 import chime
 
 
+@pytest.fixture(scope='function', autouse=True)
+def reload_chime():
+    importlib.reload(chime)
+
+
+@pytest.fixture(scope='function', autouse=True)
+def mock_pathlib_home(monkeypatch: _pytest.monkeypatch.MonkeyPatch):
+    with tempfile.TemporaryDirectory() as home_dir:
+        home_dir_path = pathlib.Path(home_dir)
+        monkeypatch.setattr(pathlib.Path, name='home', value=lambda: home_dir_path)
+        monkeypatch.setenv('APPDATA', value=str(pathlib.Path(home_dir)))
+
+
```

Comment:
Another problem can happen if the user running the test has a config file on their machine. Their config could specify another default theme, changing the theme the test run on.

`mock_pathlib_home` attempts to fix this by making a temporary directory and having it take over as the home directory (or `APPDATA` directory on Windows), where there will be no config file and chime will use the standard default theme across all tests until it is purposely changed.

Reply:
Very smart!